components/cards.css

```diff
@@ -0,0 +1,574 @@
+  width: fit-content;
+  height: 15rem;
+  margin: 3rem;
+  box-shadow: 3px 3px 5px 6px #ccc;
+  position: relative;
+  display: flex;
+}
+
+.badge-card-desc {
+  display: flex;
+  flex-direction: column;
+  /* justify-content: space-around; */
+}
+.badge-card-desc .product-name {
+  font-size: 24px;
+  font-weight: 600;
+  padding: 4px;
+}
+.badge-card-desc .product-detail {
+  font-size: 22px;
+  font-weight: 500;
+  padding: 3px;
+}
+.badge-card-desc .price {
+  font-size: 14px;
+  padding: 4px;
+}
+.card-badge-btn .btn1 {
+  margin-top: 2rem;
+  padding: 1rem;
+  font-size: 22px;
+  border: none;
+  background-color: #ff9900;
+  color: white;
+  font-weight: bold;
+  border-radius: 4px;
+}
+.card-badge-btn .btn1:hover {
+  color: black;
+  background-color: #f1c98b;
+}
+.card-badge-btn .btn2 {
+  margin-top: 2rem;
+  padding: 1rem;
+  font-size: 22px;
+  border-radius: 4px;
+  border: none;
+  background-color: #f1c98b;
+}
+.card-badge-btn .btn2:hover {
+  color: white;
+  background-color: #ff9900;
+}
+.badge {
+  background-color: rgb(119, 6, 6);
+  width: 5rem;
+  height: 2rem;
+  border-radius: 4px;
+  color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  top: 2rem;
+}
+/**************************************/
+/* Portrait - Cards */
+.portraitcards {
+  box-shadow: 3px 3px 5px 6px #ccc;
+  width: 20rem;
+  height: 40rem;
+  font-family: "Ubuntu", sans-serif;
+  margin-top: 2rem;
+}
+.portraitcards:hover {
+  border: 4px solid #f1c98b;
+}
+.portrait-card-upper-sec {
+  position: relative;
+}
+.portrait-card-upper-sec img {
+  width: 100%;
+  object-fit: cover;
+}
+.s {
+  text-decoration: line-through;
+}
+.portrait-card-upper-sec p {
+  background-color: green;
+  width: 6rem;
+  height: 3rem;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  color: white;
+  font-weight: 500;
+  font-size: 22px;
+  clip-path: polygon(100% 0%, 75% 50%, 100% 100%, 0 100%, 0 50%, 0 0);
+  position: absolute;
+  top: 0;
+  margin-top: 1rem;
+}
+.portrait-card-upper-sec i {
+  background-color: white;
+  border-radius: 50%;
+  color: black;
+  height: 4rem;
+  width: 4rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  top: 0;
+  right: 0;
+  margin-top: 1rem;
+  cursor: pointer;
+  box-shadow: 3px 3px 5px 6px #ccc;
+}
+.portrait-card-upper-sec i:hover {
+  box-shadow: 3px 3px 5px 6px #f1c98b;
+}
+.portrait-card-desc {
+  padding: 1rem;
+}
+.portrait-card-desc p {
+  margin-top: 10px;
+}
+.portrait-card-desc .large-title {
+  font-size: 32px;
+  font-weight: 600;
+}
+.portrait-card-desc .small-title {
+  font-size: 24px;
+  font-weight: 600;
+  color: rgb(153, 153, 153);
+}
+.portrait-card-desc .price {
+  font-size: 20px;
+  font-weight: 600;
+}
+.price span {
+  color: red;
+}
+.portrait-card-btn .btn1 {
+  font-size: 18px;
+  font-weight: 600;
+  padding: 12px;
+  margin-left: 10px;
+  background-color: #ff9900;
```

Comment:
background-color property is repeating at a lot of places try to declare that color in your main CSS file and use variable naming to access the color.

Reply:
@prathmeshjagtap made changes as you suggested. actually, I started using the same class for a similar component but I was making more mistakes so I stopped using the same class but will use it from now.